lib/matplotlib/backends/backend_pdf.py

```diff
@@ -484,11 +483,13 @@ def __init__(self, filename, metadata=None):
 
         self.infoDict = {
             'Creator': 'matplotlib %s, http://matplotlib.org' % __version__,
-            'Producer': 'matplotlib pdf backend%s' % revision,
+            'Producer': 'matplotlib pdf backend %s' % __version__,
```

Comment:
I thought including the version in both would help with debugging when the user has overridden just one of these. If you use Matplotlib as a component of some larger application, you might want to override Creator and leave Producer pointing to the PDF backend.